build64/la64/ldscripts/memtest_shared.lds

```diff
@@ -0,0 +1,56 @@
+OUTPUT_FORMAT("elf64-loongarch")
+OUTPUT_ARCH(loongarch);
+
+ENTRY(startup64);
+SECTIONS {
+	. = 0;
+	.text : {
+		_start = .;
+		*(.text)
+		*(.text.*)
+		*(.plt)
+		_etext = . ;
+	} = 0x9090
```

Comment:
IIUC, you want to replace that by the encoding of NOP on your architecture.

Reply:
Yes! I forgot to adjust this instruction when I porting the code, it should be `= 0x00004003`. :)
I will fix in the next commit. Thanks!